doc/helm/user-guide.md

```diff
@@ -309,6 +309,8 @@ kubectl delete -f deploy/role.yaml
 kubectl delete -f deploy/service_account.yaml
 kubectl delete -f deploy/crds/example.com_nginxes_crd.yaml
 ```
+**NOTE** Additional CR/CRD's can be added to the project by running, for example, the command :`operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=AppService`
+For more information, refer [cli][addcli] doc.
 
```

Comment:
Same issue faced with the watches for helm as well. See:

<img width="1355" alt="Screenshot 2020-04-09 at 19 17 13" src="https://user-images.githubusercontent.com/7708031/78927419-c9f23c00-7a96-11ea-918d-8f2be88d57c2.png">

The command used over the Helm Mencached sample was: $ operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=AppService

Reply:
IMO, this one is non-blocking. This doesn't change the behavior at all. Reordering of fields is a minor nuisance. The added `watchDependentResources` is a quirk of YAML marshalling due to the fact that we set `watchDependentResources = true` as a default when we unmarshal.